docs/types.md

```diff
@@ -1,8 +1,7 @@
-
-This page serves as a reference for mappings between scalar protobuf types and generated Haskell types.
-
 # Scalar Type Reference
 
+Reference for mappings between scalar protobuf types and generated Haskell types.
```

Comment:
I would prefer a full sentence here rather than just the sentence fragment.  (But the change you made to move it after the title looks correct.

Reply:
Done